lib/auth/auth_with_roles_test.go

```diff
@@ -9545,15 +9545,15 @@ func TestIsMFARequired_AdminAction(t *testing.T) {
 			name:                 "mfa verified",
 			adminActionAuthState: authz.AdminActionAuthMFAVerified,
 			expectResp: &proto.IsMFARequiredResponse{
-				Required:    false,
-				MFARequired: proto.MFARequired_MFA_REQUIRED_NO,
+				Required:    true,
```

Comment:
Do you think it's worth testing these using CreateAuthenticationChallenge instead? I've read the PR description a few times now but I confess I'm still a bit confused by it.

Reply:
`CreateAuthenticateChallenge` calls `authWithRoles.IsMFARequired` directly, so I think it's ok. I reorganized the PR description, hopefully the relation makes more sense now.